### PR TITLE
Add TDP 52 and TDP 53 & Update Headers

### PR DESCRIPTION
**Pull request by mzero2nine**
Can you please review when time permits @dimsome?

### CODE REVIEW

File: TDP/tdp-53.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+created: 2022-12-08
+---
+
+## Simple Summary
+
+This proposal requests four (4) months of operational funding for the mStable Ecosystem subDAO. The proposal will:
+
+⦁	Outline a budget proposal for the period of 1st January 2023 to 30th April 2023
+⦁	Define deliverables and methods of transparently reporting activity and performance during the funding period
+⦁	For more background on the mStable Ecosystem subDAO's purpose, mandate, structure, and internal processes; please see the initial funding request here [TDP 46](https://forum.mstable.org/t/tdp-46-mstabledao-restructure-ecosystem-subdao/895)
```
Comment (dimsome) [inline TDP/tdp-53.md:16]: consistent bullet points according to markup

Comment (dimsome) [inline TDP/tdp-53.md:16]: relative path to MIPs

File: TDP/tdp-53.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+A full itemized list of funding initiatives can be found in the “Amount Requested” portion of this Proposal. Funding received will be utilized to optimally run the subDAO to carry out its initiatives and meet its objectives.
+
+## Motivation
+
+The Ecosystem subDAO Funding Request is created in order to resolve the merger between the Community subDAO and the GrantsDAO ratified in [TDP 46](./tdp-46)
+
+## Specification
+
+**mUSD**
```
Comment (dimsome) [inline TDP/tdp-53.md:40]: H3 and all other can be h3 rather than bold

File: TDP/tdp-53.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+**Total MTA Requested: 17,351**
+
+# Deliverables
+
+Timeline: 1/23 - 4/23. This funding request will carry the Ecosystem subDAO through the first four (4) months of 2023. Monthly report comprising of highlights, lowlights, and ecosystem subDAO performance will be provided monthly to the treasury subDAO.
+
+⦁ Successful utilization of Dework, defined through the regular successful posting, quality assurance, task completion, and payout of various bounties where previously defined deliverables that add value to the mStable ecosystem are met. The ecosystem subDAO will more deeply collaborate with the builder subDAO to increase the value accrual for the mStable community via the use of Dework.
+
+⦁ Hosting of community events: Community Calls, AMA’s, Game Nights, Trivia where community members are rewarded through their participation via one or more of the following: POAP’s, NFT’s, MTA rewards, special discord roles, recognition in official announcements, and so on. Determination of the number and timing of community-based events will be coordinated with the Builder subDAO.
```
Comment (dimsome) [inline TDP/tdp-53.md:95]: bullet points style according to markup

File: TDP/tdp-53.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+- Moderator Pay: 25,000
+- Raffles - 6,250
+- Rewards for Completing Surveys - 1,875
+- Tips - 1250
+
+Summary:  83,125 MTA amount with **17,351** requested for this funding round
+
+**Total MTA Requested: 17,351**
+
+# Deliverables
```
Comment (dimsome) [inline TDP/tdp-53.md:89]: h1?

File: TDP/tdp-53.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+⦁ Hosting of community events: Community Calls, AMA’s, Game Nights, Trivia where community members are rewarded through their participation via one or more of the following: POAP’s, NFT’s, MTA rewards, special discord roles, recognition in official announcements, and so on. Determination of the number and timing of community-based events will be coordinated with the Builder subDAO.
+Accountability & Transparency
+
+It is suggested that the mStable TreasuryDAO continues to conduct a monthly review to hold the Ecosystem subDAO accountable to the deliverables defined herein.
+
+The Ecosystem subDAO commits to providing the TreasuryDAO access to information on all expenditure for each month prior to the 10th of the following month, including the category of expenditure. This will either be provided via a report/presentation.
+
+# Additional and Future Funding Requests
```
Comment (dimsome) [inline TDP/tdp-53.md:102]: h1?

Comment (mzero2nine) [reply TDP/tdp-53.md]: Fixed

**Comment by mzero2nine** [comment]
@fungalarmy and @camsoulsby please do take note of these markups and style change requests in general for future TDPs and proposals from the forum 😎

File: TDP/tdp-53.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+- Moderator Pay: 25,000
+- Raffles - 6,250
+- Rewards for Completing Surveys - 1,875
+- Tips - 1250
+
+Summary:  83,125 MTA amount with **17,351** requested for this funding round
+
+**Total MTA Requested: 17,351**
+
+## Deliverables
```
Comment (dimsome) [inline TDP/tdp-53.md:89]: This is within specifications or? should been h3?

**Review by dimsome — APPROVED** [review]

File: TDP/tdp-53.md (hunk 6 above)
Comment (mzero2nine) [reply TDP/tdp-53.md]: Yes true

File: TDP/tdp-53.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+
+* Hosting of community events: Community Calls, AMA’s, Game Nights, Trivia where community members are rewarded through their participation via one or more of the following: POAP’s, NFT’s, MTA rewards, special discord roles, recognition in official announcements, and so on. Determination of the number and timing of community-based events will be coordinated with the Builder subDAO.
+Accountability & Transparency
+
+It is suggested that the mStable TreasuryDAO continues to conduct a monthly review to hold the Ecosystem subDAO accountable to the deliverables defined herein.
+
+The Ecosystem subDAO commits to providing the TreasuryDAO access to information on all expenditure for each month prior to the 10th of the following month, including the category of expenditure. This will either be provided via a report/presentation.
+
+## Additional and Future Funding Requests
```
[the remark this answers is not in the document]
Comment (mzero2nine) [reply TDP/tdp-53.md]: 👍

**Pull request merged**